packages/analytics/analytics-metric-provider/src/components/MetricsProvider.vue

```diff
@@ -10,7 +10,7 @@ import {
   type AnalyticsBridge,
   type ExploreFilter,
   queryableExploreDimensions,
-  type QueryableExploreDimensions,
+  type QueryableExploreDimensions, type QueryDatasource,
```

Comment:
minor lint, new line probably preferred?

Reply:
Thanks -- not sure why that isn't caught by the linter.  I'd like to merge now so that I can keep rolling with the others, but I'll fix this when I fix the dashboard types later on.